docs/apis/cookies.md

```diff
@@ -99,6 +99,15 @@ As of iOS 14, you cannot use 3rd party cookies by default. Add the following lin
 </array>
 ```
 
+Adding the ```WKAppBoundDomains``` key to ```Info.plist``` causes all WKWebView instances in the application to be restricted however. Access to the following APIs will be denied:
+
+- JavaScript injection
+- custom style sheets
+- cookie manipulation
+- message handlers
+
+To re-enable these APIs, set the "ios.limitsNavigationsToAppBoundDomains" in your ```capacitor.config.ts``` to ```true```.
```

Comment:
This is true but not quite complete - if you add `WKAppBoundDomains` you **must** set `ios.limitsNavigationToAppBoundDomains` to avoid issues - [CAPInstanceDescriptor.h#L123](https://github.com/ionic-team/capacitor/blob/4a02e7a2720ca5fa0f7f947ddf45c50fa4bc1ebf/ios/Capacitor/Capacitor/CAPInstanceDescriptor.h#L123)

We should just add a line saying that you must set `ios.limitsNavigationsToAppBoundDomains` to be true.

Reply:
Thanks @markemer for looking into this. I updated / shortened the docs